dhall/src/Dhall/Normalize.hs

```diff
@@ -891,7 +896,7 @@ isNormalized e0 = loop (Syntax.denote e0)
       ToMap x t -> case x of
           RecordLit _ -> False
           _ -> loop x && all loop t
-      Field r k -> case r of
+      Field r (Syntax.fieldAccessLabel -> k) -> case r of
```

Comment:
Since normalization turns all the `Maybe s` fields into `Nothing`, I think we need to check that these fields are `Nothing`. We should do the same for `RecordField` too.